.github/workflows/ci.yml

```diff
@@ -60,12 +60,17 @@ jobs:
       - name: Get go version
         id: go-version
         run: echo "name=version::$(go env GOVERSION)" >> $GITHUB_OUTPUT
-      - name: Docker Login
-        env:
-          DOCKER_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
-          DOCKER_PASSWORD: ${{ secrets.DOCKERHUB_PASSWORD }}
-        run: |
-          echo "${DOCKER_PASSWORD}" | docker login --username "${DOCKER_USERNAME}" --password-stdin
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GORELEASER_GITHUB_TOKEN }}
```

Comment:
seems the docs for `docker/login-action` recommends using `secrets.GITHUB_TOKEN` ([source](https://github.com/docker/login-action#github-container-registry)) which is autogenerated and has write permission on packages by default ([source](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token))